tgstation.dme

```diff
@@ -5384,8 +5384,8 @@
 #include "code\ze_genesis_call\genesis_call.dm"
 #include "interface\interface.dm"
 #include "interface\menu.dm"
-#include "interface\skin.dmf"
 #include "interface\stylesheet.dm"
+#include "interface\skin.dmf"
```

Comment:
```suggestion
#include "interface\skin.dmf"
#include "interface\stylesheet.dm"
```
This keeps trying to change itself and the code doesnt like when its changed so we have to make sure it doesnt change itself 💀